wiki/Performance_Points/en.md

```diff
@@ -1,176 +1,15 @@
 ---
```

Comment:
this article was requested to be cleaned-up, not be removed and split-up in separate sections. #3781 is already handling this